journals/apps/api/v1/views.py

```diff
@@ -62,3 +64,31 @@ def _revoke_access(self, order_number):
                     access_record.journal,
                     access_record.order_number)
         return HttpResponse()
+
+
+class UserPageVisitViewSet(viewsets.ModelViewSet):
+    """API for UserPageVisit model"""
+    lookup_field = 'user_id'
+    queryset = UserPageVisit.objects.all()
+    serializer_class = UserPageVisitSerializer
+    permission_classes = (IsAdminUser,)
```

Comment:
why we have `IsAdminUser` permission requirement here?

Reply:
will probably want this to check that it's a logged in journal user